srcpkgs/LuaJIT/patches/enable-debug-symbols.patch

```diff
@@ -1,16 +1,8 @@
-From: =?utf-8?q?Ond=C5=99ej_Sur=C3=BD?= <ondrej@sury.org>
-Date: Tue, 17 Nov 2015 16:27:11 +0100
-Subject: Enable debugging symbols in the build
-
----
- src/Makefile | 4 ++--
- 1 file changed, 2 insertions(+), 2 deletions(-)
-
-diff --git src/Makefile src/Makefile
-index 8a38efd..6b73a89 100644
+diff --git a/src/Makefile b/src/Makefile
+index 3a6a432..b606927 100644
 --- a/src/Makefile
 +++ b/src/Makefile
-@@ -54,9 +54,9 @@ CCOPT_arm64=
+@@ -53,9 +53,9 @@ CCOPT_arm64=
  CCOPT_ppc=
  CCOPT_mips=
  #
```

Comment:
As you mentioned its probably possible to leave this up to xbps-src to add this flag when building `-dbg`

debug flag `-g` is added to CFLAGS automatically by xbps-src, see `common/xbps-src/shutils/common.sh`

Reply:
yeah I kinda just grandfathered this path without thinking about it too much, but the debug symbols do indeed generate without it